libcxx/include/__ranges/drop_view.h

```diff
@@ -90,6 +90,10 @@ namespace ranges {
       requires (!(__simple_view<_View> &&
                   random_access_range<const _View> && sized_range<const _View>))
     {
+      if constexpr (random_access_range<const _View> && sized_range<const _View>) {
+        auto __dist = ranges::distance(__base_);
+        return ranges::begin(__base_) + std::min<range_difference_t<_View>>(__count_, __dist);
```

Comment:
Btw, please double check if this file has undef the min/max macro on the top

Reply:
Not found anything similar. Besides, `std::min` is also used in other places of this file.

Reply:
yes there is already push/pop macros in this file